examples/examplepb/a_bit_of_everything.proto

```diff
@@ -32,6 +32,68 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
   consumes: "application/x-foo-mime";
   produces: "application/json";
   produces: "application/x-foo-mime";
+  security_definitions: {
+    security: {
+      key: "BasicAuth";
+      value: {
+        type: TYPE_BASIC;
+      }
+    }
+    security: {
+      key: "ApiKeyAuth";
+      value: {
+        type: TYPE_API_KEY;
+	in: IN_HEADER;
+	name: "X-API-Key";
+      }
+    }
+    security: {
+      key: "OAuth2";
+      value: {
+        type: TYPE_OAUTH2;
+	flow: FLOW_ACCESS_CODE;
+	authorization_url: "https://example.com/oauth/authorize";
+	token_url: "https://example.com/oauth/token";
+	scopes: {
+	  scope: {
+	    key: "read";
+	    value: "Grants read access";
+	  }
+	  scope: {
+	    key: "write";
+	    value: "Grants write access";
+	  }
+	  scope: {
+	    key: "admin";
+	    value: "Grants read and write access to administrative information";
+	  }
+	}
+      }
+    }
+  }
+  security: {
+    security_requirement: {
```

Comment:
I don't see an entry for bearer (in authorization header). Which one would that map to? Reading the docs I guess an apiKey with the header "authorization"?